lib/cloud_controller/paging/sequel_paginator.rb

```diff
@@ -14,8 +15,7 @@ def get_page(sequel_dataset, pagination_options)
 
       sequel_dataset = sequel_dataset.order(sequel_order)
       sequel_dataset = sequel_dataset.order_append(Sequel.asc(Sequel.qualify(table_name, :guid))) if sequel_dataset.model.columns.include?(:guid)
-
-      records, count = if can_paginate_with_window_function?(sequel_dataset)
+      records, count = if !(EXCLUDE_FROM_PAGINATION_WITH_WINDOW_FUNCTION.include? table_name) && can_paginate_with_window_function?(sequel_dataset)
```

Comment:
The line above contains `.include?(...)` (i.e. with parentheses). Although it's a matter of taste which syntax to use, I would prefer consistency (at least in the same file).

The surrounding parentheses are not needed:
```
!EXCLUDE_FROM_PAGINATION_WITH_WINDOW_FUNCTION.include?(table_name)
```